libs/widgets/labelCheckerDialog.py

```
@@ -0,0 +1,424 @@
+            self._update_summary()
+
+            issue_count = len(self.issues)
+            label_count = len(labels_with_files)
+            self.status_label.setText(
+                f"Found {issue_count} issues in {label_count} unique labels"
+            )
+
+            self.fix_selected_btn.setEnabled(issue_count > 0)
+            self.export_btn.setEnabled(issue_count > 0)
+
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)